bin/index.js

```diff
@@ -104,50 +80,24 @@ const askQuestions2 = () => {
       type: 'list',
       name: '_actionType',
       message: 'What action is the certificate asserting?',
-      choices: ["REGISTER", "UPDATE"],
+      choices: ["REGISTER", "DE_REGISTER"],
       default: "REGISTER"
     },
     {
-      name: "_releaseNumber",
-      type: "input",
-      message: "What's the release number?",
-      default: '0.0.1'
-    },
-    {
-      name: "_releaseName",
+      name: "_comment",
       type: "input",
-      message: "What's the release name?",
-      default: 'My first release'
+      message: "Comment your change (optional):"
     },
-    // {
-    //   type: 'list',
-    //   name: '_offChainStorage',
-    //   message: 'Where are you storing offchain metadata?',
-    //   choices: ["CIP-26", "IPFS", "GITHUB", "BITBUCKET"],
-    //   default: "GITHUB"
-    // },
     {
       name: "_offChainStoragePath",
```

Comment:
Please, rename also this variable to e.g.
```suggestion
      name: "_metadataUrl",
```

Reply:
Fixed on 79fbe06